petasus/scripts/sage2lib.py

```diff
@@ -24,7 +24,11 @@ def read_peptides(peptides, qvalue):
     polars.DataFrame
         The parsed and filtered peptides.
     """
-    peptide_df = pl.read_csv(peptides, separator="\t")
+    try:
```

Comment:
Request: Could we add an error/warning message saying the fallback is occurring?